phpBB/adm/style/acp_styles.html

```diff
@@ -32,6 +32,10 @@
 
 <!-- IF L_EXPLAIN --><p>{L_EXPLAIN}</p><!-- ENDIF -->
 
+<fieldset class="quick">
+	<span class="small"><a href="https://www.phpbb.com/customise/db/styles-2" target="_blank">{L_BROWSE_STYLES_DATABASE}</a></span>
```

Comment:
Yeah that's support though. Are those IST Titania's mirrors or independent? Because phpBB only condones downloading from texts and styles from Titania ;)